src/main/java/nl/novi/techiteasy/models/Television.java

```diff
@@ -0,0 +1,216 @@
+package nl.novi.techiteasy.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "type")
+    private String type;
+
+    @Column(name = "brand")
+    private String brand;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "price")
+    private Double price;
+
+    @Column(name = "available_size")
```

Comment:
Ook hier vul je exact dezelfde naam in die al als default gemaakt wordt. Hibernate vervangt namelijk camelcase automatisch met snakecase